modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -239,6 +238,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           '#langcode' => $item->getLangcode(),
         ],
         '#link' => $link_render_array,
+        '#link_uri' => $link_url,
```

Comment:
```suggestion
        '#link_url' => $link_url,
```
Potentially update to suggest this is an URL rather than a URI.